test-clients/src/main/java/com/hedera/services/bdd/suites/perf/SubmitMessageLoadTest.java

```diff
@@ -203,7 +204,7 @@ private static HapiApiSpec runSubmitMessages() {
 						INVALID_TOPIC_ID,
 						INSUFFICIENT_PAYER_BALANCE)
 				.hasKnownStatusFrom(SUCCESS, OK, INVALID_TOPIC_ID, INSUFFICIENT_PAYER_BALANCE
-						,UNKNOWN,TRANSACTION_EXPIRED)
+						,UNKNOWN,TRANSACTION_EXPIRED, MESSAGE_SIZE_TOO_LARGE)
```

Comment:
Why is `MESSAGE_SIZE_TOO_LARGE` expected ?

Reply:
For this load test, I have changed the message size to be "base size + variable size". If the JRS test put a combination like for base size 768 + variable size 512. Which means the final generated message can have sizes anywhere from 768 to 1280, that may exceed the default allowed message size of 1024. 
Putting this response code is a just in case precaution.